forester/scripts/v2_stats.py

```
@@ -0,0 +1,214 @@
+    parser.add_argument('logfile', nargs='?', default='-', help='Log file to analyze')
+    parser.add_argument('--tree-type', help='Filter to specific tree type')
+    
+    args = parser.parse_args()
+    
+    analyzer = V2TpsAnalyzer()
+    
+    # Read and parse log file
+    if args.logfile == '-':
+        log_file = sys.stdin
+    else:
+        log_file = open(args.logfile, 'r')
```

Ruff (0.11.9)

Use ternary operator `log_file = sys.stdin if args.logfile == '-' else open(args.logfile, 'r')` instead of `if`-`else`-block

Replace `if`-`else`-block with `log_file = sys.stdin if args.logfile == '-' else open(args.logfile, 'r')`

(SIM108)